test/e2e/specs/project.edit.spec.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+describe('Login', function () {
+	const loginPage = require('../pages/login.page');
+	const dashboardPage = require('../pages/dashboard.page');
+	const requestManager = require('../../api/request.manager');
+	const projectSetting = require('../pages/projects/setting.page');
+	const common = require('../pages/common');
+	let id;
+
+	beforeAll(function*() {
+		browser.waitForAngularEnabled(false);
+		browser.get(`${browser.params.baseUrl}/signin`);
```

Comment:
@AT-04/at-04 
keep in mind the layers in the framework
you should not use protractor code in the specs.